MHS/README.md

```diff
@@ -0,0 +1,67 @@
+# Message Handling System
+
+The MHS Adaptor implements a messaging standard called the External Interface Specification, which defines in some detail a number of patterns for transport layer communication with the NHS Spine. The intent of this MHS Adaptor is to hide this implementation detail from the supplier, and so make it easier to connect to Spine and perform business operations such as interacting with Spine services like PDS.
+The MHS adaptor is composed of three main services, coloured in orange, which are executed in Docker containers:
+1.	The MHS Outbound Service which is responsible for listening for requests from the wider local system context and transmitting these to Spine
+2.	Spine Route Lookup, which is used to lookup routing and reliability information from Spine's directory service.
+3.	the MHS Inbound Service which is responsible for listening for incoming requests from Spine.
+This Link provides a view of the services and Python modules which make up the MHS Adaptor based on the AWS Architecture Exemplar.  ***Remove???? Or simpify***
+The Integration-Adaptors MHS Github repo has significant details about this Adaptor including API Documentation, example Postman collection requests, AWS and Azure Deployment Exemplars and Terraform resources that you may find useful.  
+
+
+
+##Onboarding
```

Comment:
need spaces after the hashes, formatting hasn't been applied

Reply:
done